README.md

```diff
@@ -1,20 +1,30 @@
-# PROJECT NAME
-GIT BASED MICROVERSE PROJECT
-
-
-<!-- #TASK
-
-
-THE FOLLOWING TASKS WERE EXECUTED
-document your work in a professional way.
-Create a new GitHub repo.
-Create a new branch according to the GitHub flow rules.
-Add a .gitignore file.
-Set up a linter for HTML and CSS and make sure that any files that are not relevant are ignored by git.
-Add descriptive README file to your project - please use this template but remember to customize it to your project.
-Add 3 files (each one in a separate commit):
-A .html file that includes a header with the text "Hello Microverse!". Use h1 tag for that.
-A .css file that includes one class with styles for your header (make it in your favorite color).
-A test.md file with the text "This file should be ignored by git".
-Make sure that the last file is actually ignored by git and is not present in your GitHub repository.
-Open a pull request. -->
\ No newline at end of file
+# Project Name
+Learning basic Git/GitHub/GitHub Flow. screenshot
+
+Additional description about the project and its features.
+
+# Built With
+HTML
+CSS
+Getting Started
+This is my first project in Microverse. It's is aimed at learing basic Git/GitHub/GitHub Flow
+
+# Authors
+👤 Author
+
+GitHub: @amklass11
+LinkedIn: Amk lass
+🤝 Contributing
+#Contributions, issues, and feature requests are welcome!
+
+Feel free to check the issues page.
+
+#Show your support
+Give a ⭐️ if you like this project! and cheers✨
+
+ # Acknowledgments
+Microverse offered guidance on how to make the project more professional
+
+
+# 📝 License
+This project is MIT licensed.
```

Comment:
[REQUIRED] As mentioned by the previous CR, please kindly make sure, your [README.md](https://github.com/amklass11/hello-microverse-/tree/lass#readme) has all sections that are presented in this [template](https://github.com/microverseinc/readme-template) and all sections are properly adjusted to your project.